keps/sig-apps/3329-retriable-and-non-retriable-failures/kep.yaml

```diff
@@ -5,7 +5,7 @@ authors:
 owning-sig: sig-apps
 participating-sigs:
   - sig-scheduling
-status: implementable
+status: implemented
```

Comment:
We don't mark KEPs as implemented until they reach GA.

Reply:
thanks for checking, reverted